department-of-veterans-affairs/va.gov-cms
# VACMS-3825: Add manilla option to date fields.

## Description
See #3825 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/102628000-080c2900-4117-11eb-8650-bbc51316afae.png)

## QA steps
As an administrator, go to `/node/add/event` and visually verify that Manilla is available for timezone option.

### Definition of Done


### Select Team for PR review


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
